### DISCUSSION TITLE
Language Selector for individual cfp entries doesn't show up

### DISCUSSION THREAD

**Discussion by Findus23** [pretalx/pretalx#1352] [Q&A] [answered]
I have been wondering about this far too long and I am assuming I am doing something wrong, so I will ask here:

Last year we organized a successfull event thanks to Pretalx. Now this year we want to repeat this, so I created a new event in pretalx, set all settings to the same thing and opened the cfp. But there seems to be one difference:

In last year's cfp entries, there is a dropdown where one can select the language of a talk (and this one is then shown in the UI, API responses, etc.)
![image](https://user-images.githubusercontent.com/6266037/189855082-b0c8ef39-ed5c-400e-98f0-d861b4f30eef.png)

But when I look at the cfp entries for this year, this selector doesn't show up:
![image](https://user-images.githubusercontent.com/6266037/189855458-a512a579-c28e-4dd6-b8cc-6dd544a76033.png)

The code for this looks really straight-forward:

https://github.com/pretalx/pretalx/blob/v2.3.1/src/pretalx/orga/templates/orga/cfp/text.html#L90-L97
https://github.com/pretalx/pretalx/blob/01bfe30e2185215f3f89626efe6d305afc1e1396/src/pretalx/orga/templates/orga/cfp/text.html#L90-L97
https://github.com/pretalx/pretalx/blob/v2.3.1/src/pretalx/event/models/event.py#L400-L408
https://github.com/pretalx/pretalx/blob/01bfe30e2185215f3f89626efe6d305afc1e1396/src/pretalx/event/models/event.py#L400-L408

And the database entries are also the same:
```text
pretalx=# select name,locale,locale_array from event_event;
           name            | locale | locale_array 
---------------------------+--------+--------------
 {"en": "MatomoCamp 2021"} | en     | en,de,fr
 {"en": "MatomoCamp"}      | en     | en,de,fr
```

So I am running out of ideas about what I could be doing wrong this year.

Do you have a guess?

I am using v2.3.1 on a self-hosted server.

**Comment by rixx** [comment, marked as answer]
The organiser area, in an attempt to avoid overwhelming you with information, only shows fields that are relevant to the current event. I'd assume that in your CfP settings, you have set the language to "Do not ask", and so pretalx assumes that you don't care about proposal/session language, and doesn't show the drop-down.

**Reply by Findus23** [reply]
Many thanks, that worked. It makes sense to not show all options, but I assumed the setting would only affect cfp users. Maybe this sentence could be extended a bit:
> Select which information should be requested and/or required during CfP proposal. 

Weirdly enough for last year's event it is also set to "Do not ask", but I can't promise I haven't changed that after the event.

**Reply by Findus23** [reply]
But this leads me to a related question (I won't create a new discussion for it): 
Is there a way to set the language of a talk to a language that Pretalx is not translated in. It would be fine not showing the user interface in that language, but just show the language of that specific talk as e.g. Italian.

**Reply by rixx** [reply]
Support for that kind of language selection is being tracked in #392 . There is a PR, however it is unfinished and has accumulated a lot of (serious) merge conflicts. I'd be happy to have that feature in pretalx, but prioritising it without community contributions or financial support is going to take a while, I'm afraid.
